modules/manually-create-identity-access-management.adoc

```diff
@@ -47,7 +47,7 @@ compute:
 ----
 <1> This line is added to set the `credentialsMode` parameter to `Manual`.
 endif::aws[]
-. Run the {product-title} installer to generate manifests:
+. Run the following command from the directory that contains the installation program, to generate the manifests:
```

Comment:
comma after program not needed
Or you could go with:

> To generate the manifests, run the following command from the directory that contains the installation program:

Reply:
Thank you for highlighting this. I have gone with your suggestion. Resolved.